nemo/collections/asr/parts/preprocessing/segment.py

```diff
@@ -183,7 +171,7 @@ def from_file(
                 logging.error(f"Loading {audio_file} via pydub raised CouldntDecodeError: `{err}`.")
 
         if samples is None:
-            libs = "soundfile, kaldiio, and pydub" if HAVE_KALDI_PYDUB else "soundfile"
+            libs = "soundfile, kaldiio, and pydub" if HAVE_PYDUB else "soundfile"
```

Comment:
"soundfile, kaldiio, and pydub" -> "soundfile, and pydub"
Need to drop kaldio here.

Reply:
Fixed